ssz-rs/src/bitlist.rs

```diff
@@ -159,14 +161,22 @@ impl<const N: usize> Deserialize for Bitlist<N> {
         }
 
         let (last_byte, prefix) = encoding.split_last().unwrap();
+        if *last_byte == 0u8 {
```

Comment:
let's add a test for this case explicitly, as a trade off b/t not using `checked_sub` here but also not losing this invariant if someone changed this code

Reply:
Added a case to the [`decode_bitlist` test](https://github.com/ralexstokes/ssz-rs/pull/57/files#diff-7424acc2e06bfc6ea7bd3686c0618f6541553fca52fb1c71149a08ad0e93c52bR288)